swift/extractor/infra/SwiftDispatcher.h

```diff
@@ -78,7 +78,7 @@ class SwiftDispatcher {
     waitingForNewLabel = e;
     visit(e);
     if (auto l = store.get(e)) {
-      if constexpr (!std::is_base_of_v<swift::TypeBase, E>) {
+      if constexpr (std::is_base_of_v<LocatableTag, TrapTagOf<E>>) {
```

Comment:
I think this was necessary at some stage of my local changes, but it is not any more. It still makes more sense to make the test about being locatable rather than not being a type, but I've actually already included that in [another PR](https://github.com/github/codeql/pull/9818) where this was necessary, so I'll revert it here